src/utils.ts

```diff
@@ -219,7 +219,7 @@ export class Utils {
     if (typeof val === 'string') {
       if (val === 'auto' || val === '') h = 0;
       else {
-        let match = val.match(/^(-[0-9]+\.[0-9]+|[0-9]*\.[0-9]+|-[0-9]+|[0-9]+)(px|em|rem|vh|vw|%)?$/);
+        let match = val.match(/^(-[0-9]+\.[0-9]+|[0-9]*\.[0-9]+|-[0-9]+|[0-9]+)(px|em|rem|vh|vw|%|cm|mm)?$/);

```

Comment:
do browser support cm ? and how would I convert this to px to do match (which I do for rem/em)

Reply:
Browsers do support cm/mm. You can see the spec here: https://www.w3.org/TR/css-values-3/#absolute-lengths
There's also a conversion rate given for cm to px. I updated the `getCellHeight` method to accommodate for cm/mm units